src/Files.Package/Files.Package.wapproj

```diff
@@ -100,6 +100,12 @@
     <ProjectReference Include="..\Files.FullTrust\Files.FullTrust.csproj" />
   </ItemGroup>
   <ItemGroup>
+    <Content Include="7z.dll">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
```

Comment:
Does this work with the MIT license?

Reply:
I'm no lawyer but yeah it does. 7z.dll is distributed with LGPG license which is compatible with MIT.
This from 7zip website:
> Can I use the EXE or DLL files from 7-Zip in a Commercial Application?
Yes, but you are required to specify in your documentation (1) that you used parts of the 7-Zip program, (2) that 7-Zip is licensed under the GNU LGPL license and (3) you must give a link to www.7-zip.org, where the source code can be found.

Reply:
We can it to this page
![image](https://user-images.githubusercontent.com/39923744/182854951-99e74a86-996e-4564-a1f9-4bbe4165a10b.png)